container/config.ini

```diff
@@ -18,7 +18,7 @@ cpe-oval = /usr/share/openscap/cpe/openscap-cpe-oval.xml
 ssg = /usr/share/xml/scap/ssg/content
 
 [CVEScanner]
-fetch-cve = yes
+fetch-cve = no
```

Comment:
I think it would be beneficial for upstream community to fetch new CVE data by default.